code/game/machinery/computer/arcade/orion.dm

```diff
@@ -540,7 +540,7 @@ GLOBAL_LIST_INIT(orion_events, generate_orion_events())
 	sleep(3 SECONDS)
 	say("Oh, God! Code Eight! CODE EIGHT! IT'S GONNA BL-")
 	playsound(loc, 'sound/machines/buzz-sigh.ogg', 25, TRUE)
-	sleep(3.6)
+	sleep(.36 SECONDS)
```

Comment:
bro stop

Reply:
literally this is a compile issue

Reply:
This was fixed in 062744d78a6cf80be4c531e32b8f5956c1fb4310, what commit are you reading?

Reply:
the most recent one